te_markdown/define-password.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: page
+title: "Tutorial: Define Passwords"
+permalink: /te_markdown/define-passwords/
+---
+
+Estimated reading time: 5 minutes
+
+## Motivation
+
+Often, applications under test require user credentials like username and password for user input. In order not to store user information directly in the test case descriptions in plain text, viewable for all test participants, the Test-Editor offers the possibility to make these data available to the test cases with following approaches. 
+
+
+* The tester can specify variables inside the test case (variables must exist as environment variables in the test environment).
+
+How to define these environment variables so that during the execution of test cases they can be used, is described in the following chapter.
```

Comment:
"How to define these environment variables so that during the execution of test cases they can be used, is described in the following chapter." -> "How to define these environment variables so that they may be used during the execution of test cases is described in the following chapter."